### ISSUE TITLE
Landing Page country highlights are not AA color accessibility compliant

### ISSUE THREAD

**Issue by elioqoshi** [ooni/explorer#315]
## Expected Behavior

The contrast ratio for normal text needs to be at least 4.5 to be WCAG AA compliant.

## Actual Behavior

The contrast ratio for normal text needs to be at least 4.5 which is not the case in the country highlight cards.

## Steps to Reproduce the Problem

Inspect the text on the country highlight cards with the accessibility inspector tool to identify the color contrast ration.

![image](https://user-images.githubusercontent.com/5436686/64852975-be32ba00-d61a-11e9-80fa-dae9a6c5b076.png)

**Comment by hellais** [comment]
@elioqoshi what colors do you suggest as an alternative?

**Comment by elioqoshi** [comment]
@hellais you can input the current color and this tool gives you the closest accessible color code:
https://contrast-finder.tanaguru.com/result.html?foreground=%23249F40&background=%23FFFFFF&ratio=4.5&isBackgroundTested=false&algo=Rgb&distanceSort=asc

In the case of Iran, the green would become #3C8551

**Comment by hellais** [comment]
I see now that the colors of these tiles are actually not taken from our design system. I wonder what the rational behind that was cc @holantonela.

Maybe it's wise to improve these by picking some colors which come from the color palette in our design system (see: design.ooni.io/colors).

FYI here is where all the colors are defined: https://github.com/ooni/explorer/blob/master/components/landing/highlights.json

**Comment by holantonela** [comment]
Thanks Elio for reporting this! I inspected the entire site with [Firefox's a11y tool]( https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector) (which is very cool tbh), and I found critical elements failing on color contrast tests, [like](https://user-images.githubusercontent.com/7602174/66670622-9b52f000-ec30-11e9-8b98-a6cbe6dc955d.png) [menu](https://user-images.githubusercontent.com/7602174/66670624-9b52f000-ec30-11e9-9b0f-8ae00f6612c8.png) [items](https://user-images.githubusercontent.com/7602174/66670626-9beb8680-ec30-11e9-9734-bbfe93340b1b.png) [or](https://user-images.githubusercontent.com/7602174/66670627-9beb8680-ec30-11e9-803c-5bddb5705131.png) [titles](https://user-images.githubusercontent.com/7602174/66670629-9beb8680-ec30-11e9-946a-d15ae34076d0.png). 

I'm wondering if is there a practical way to improve those issues from the main CSS/SASS/JS styleguide. I'm reading about it for Tor's styleguide too. For instance, Bootstrap has its own [features and limitations](https://getbootstrap.com/docs/4.0/getting-started/accessibility/).

I agree with @hellais; we can revisit the [OONI styleguide](https://design.ooni.io/colors/) and plan an iteration that contemplates different font-sizes and also better contrast for picked colors. What do you think is the best approach?

Should I file a new ticket or let's have this convo here? @elioqoshi @hellais

**Comment by hellais** [comment]
I think maybe we can file a new issue on the design-system repo to make these accessibility related improvements, but also keep this issue open so that we don't forget to update the colors of the cards on the Explorer landing page too.

See: https://github.com/ooni/design-system/issues/70
